README.md

```diff
@@ -114,6 +115,15 @@ Alternative: if you don't want to change your PATH, you can set ONNX_PROTOC_EXEC
 set CMAKE_ARGS=-DONNX_PROTOC_EXECUTABLE=<full_path_to_protoc.exe>
 ```
 
+Before proceeding, ensure that you have pybind11 installed on your computer. You can do this using pip:
+```bat
+pip install pybind11
+```
+Alternatively, you can use conda-forge:
+```bat
+conda install -c conda-forge pybind11
+```
+
```

Comment:
Is manual installation for pybind11 really needed? ONNX has its own pybind11 as submodule. Could you please try to build ONNX from source without manual installation for pybind11? Please let me know if you bump into any issues. Thanks!

Reply:
When I built ONNX from source during my very first installation, I ran into the following issue:

"could not find pybind11 missing: pybind11_dir"

So I assumed the manual installation of pybind11 was necessary. I can try building ONNX from source without this installation again, though!

Reply:
Please run `git submodule update --recursive` before building ONNX from source if you didn't include `--recursive` while git clone previously.

Reply:
It seems to work this time! It appears that there were multiple versions of some packages in my conda environment, which caused some issues. I can change this to say, "If you receive an error, check the following:" instead of suggesting a manual installation. Thanks!